### PR TITLE
[wallet-ext] - add % stake share to validator list

### PR DESCRIPTION
**Pull request by mamos-mysten**
image

displays % stake share on validator list view.

### CODE REVIEW

File: apps/wallet/src/shared/formatting.ts [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
 import type { FormatNumberOptions } from 'react-intl';
 
 export const percentageFormatOptions: FormatNumberOptions = {
     style: 'percent',
     maximumFractionDigits: 2,
     minimumFractionDigits: 0,
 };
+
+export const formatPercentage = (
+    num1: bigint | string | number,
```
Comment (Jordan-Mysten) [inline apps/wallet/src/shared/formatting.ts:15]: Very unclear to me what these arguments are supposed to represent. Probably better to use terms like denominator and numerator.

File: apps/wallet/src/shared/formatting.ts [hunk 2]
```diff
@@ -1,10 +1,27 @@
...
+    num2: bigint | string | number,
+    decimalPlaces = 3
+) => {
+    const bn1 = new BigNumber(num1.toString());
+    const bn2 = new BigNumber(num2.toString());
+    const percentage = bn1
+        .div(bn2)
+        .multipliedBy(100)
+        .decimalPlaces(decimalPlaces)
+        .toNumber();
```
Comment (Jordan-Mysten) [inline apps/wallet/src/shared/formatting.ts:25]: Why toNumber it? Why not just `toFormat`?

**Review by Jordan-Mysten — APPROVED** [review]
Nits but LGTM

File: apps/wallet/src/shared/formatting.ts (hunk 2 above)
Comment (mamos-mysten) [reply apps/wallet/src/shared/formatting.ts:25]: needed to be a number for sorting... probably doesn't make sense as a formatting util, so I just moved it into a targeted util for staking

**Pull request merged**